README_comments.md

```diff
@@ -0,0 +1,247 @@
+```bash
+bash tests/test_integration_workflow_with_conda/test.local.sh
+```
+Execute one of the following commands to run the test workflow 
+on a [Slurm][slurm]-managed high-performance computing (HPC) cluster:
+
+* Test workflow with **Singularity**:
+
+```bash
+bash tests/test_integration_workflow/test.slurm.sh
+```
+* Test workflow with **Conda**:
+
+```bash
+bash tests/test_integration_workflow_with_conda/test.slurm.sh
+```
+
+> **NOTE:** Depending on the configuration of your Slurm installation you may
+> need to adapt file `slurm-config.json` (located directly under `profiles`
+> directory) and the arguments to options `--cores` and `--jobs`
+> in the file `config.yaml` of a respective profile.
+> Consult the manual of your workload manager as well as the section of the
+> Snakemake manual dealing with [profiles].
+
+# Running the workflow on your own samples
+
+1. Assuming that your current directory is the repository's root directory,
+create a directory for your workflow run and move into it with:
+
+    ```bash
+    mkdir config/my_run
+    cd config/my_run
+    ```
+
+2. Create an empty sample table and a workflow configuration file:
+
+    ```bash
+    touch samples.tsv
+    touch config.yaml
+    ```
+
+3. Use your editor of choice to populate these files with appropriate
+values. Have a look at the examples in the `tests/` directory to see what the
+files should look like, specifically:
+
+    - [samples.tsv](tests/input_files/samples.tsv)
+    - [config.yaml](tests/input_files/config.yaml)
+
+    - For more details and explanations, refer to the [pipeline-documentation]
+
+
+4. Create a runner script. Pick one of the following choices for either local
+or cluster execution. Before execution of the respective command, you need to
+remember to update the argument of the `--singularity-args` option of a
+respective profile (file: `profiles/{profile}/config.yaml`) so that
+it contains a comma-separated list of _all_ directories
+containing input data files (samples and any annotation files etc) required for
+your run.
+
+    Runner script for _local execution_:
+
+    ```bash
+    cat << "EOF" > run.sh
+    #!/bin/bash
+
+    snakemake \
+        --profile="../profiles/local-singularity" \
+        --configfile="config.yaml"
+
+    EOF
+    ```
+
+    **OR**
+
+    Runner script for _Slurm cluster exection_ (note that you may need
+    to modify the arguments to `--jobs` and `--cores` in the file:
+    `profiles/slurm-singularity/config.yaml` depending on your HPC
+    and workload manager configuration):
+
+    ```bash
+    cat << "EOF" > run.sh
+    #!/bin/bash
+    mkdir -p logs/cluster_log
+    snakemake \
+        --profile="../profiles/slurm-singularity" \
+        --configfile="config.yaml"
+    EOF
+    ```
+
+    When running the pipeline with *conda* you should use `local-conda` and
+    `slurm-conda` profiles instead.
+
+5. Start your workflow run:
+
+    ```bash
+    bash run.sh
+    ```
+
```

Comment:
A few words about what the user expects here? E.g. If the workflow starts correctly, you will see messages about its progress on the screen. If the workflow finishes successfully, your results will be in...